infrastructure/server/oidc_idp_bcsc.tf

```diff
@@ -24,7 +24,6 @@ resource "aws_cognito_identity_provider" "dev_bcsc_oidc_provider" {
   }
 
   attribute_mapping = {
-    username = "sub",
```

Comment:
remove the username sub mapping from the BCSC IDP mapping, cause we don't have this in the IDIR and BCEID config file, and Cognito auto adds it seems like